TGC/history/units/FIL_oob.txt

```diff
@@ -0,0 +1,41 @@
+army = {
```

Comment:
Is this tag supposed to exist at game start? If not I don't think we'll need this as sadly the Vic 2 engine can't spawn units with OOBs like newer Paradox games can.